lib/js/components/TextGroup/TextGroup.vue

```diff
@@ -206,33 +212,18 @@ $text-group-colors: (
 	&.-ds-loading {
 		pointer-events: none;
 
-		#{$self}__eyebrow {
-			height: 11px;
-			margin: 1px 0 2px 0;
-		}
-
-		#{$self}__main {
-			height: 16px;
-			margin: 1px 0;
-		}
-
+		#{$self}__eyebrow,
+		#{$self}__main,
 		#{$self}__supporting {
-			height: 12px;
-			margin: 2px 0;
+			align-items: center;
+			display: flex;
+			height: 1.25em; // Fallback for older browsers
```

Comment:
if we could move this styles out of `.-ds-loading` then height won't be jumping on loading state on old browser - it'll be little off for all states

Reply:
Leaving as it is, otherwise we would need to apply this height only if ellipsis is enabled